debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouter.java

```diff
@@ -56,6 +56,9 @@
     private String fieldSchemaVersion;
 
     private String routeByField;
+    private Boolean routeTombstoneOnEmptyPayload;
+
+    private Boolean isDeleteEvent = false;
```

Comment:
Is this needed a field, or can it just be a local var?

Reply:
Yeah this can definitely be a local var